cpp/src/IceGrid/ReapThread.cpp

```diff
@@ -176,40 +169,30 @@ ReapThread::add(
 
     if (timeout > 0s)
     {
-        //
-        // If there is a new minimum wake interval then wake the reaping
-        // thread.
-        //
+        // If there is a new minimum wake interval then wake the reaping thread.
```

Comment:
Why does `calcWakeInterval` have to iterate through all sessions? Can't we simply compare the timeout of the session we are adding with the current wake interval? Seems this would be only required when we remove a session to find the new minimum.

Reply:
I left this code as is, just reformatted the comment.